src/mpi/messages.rs

```diff
@@ -255,11 +255,11 @@ impl Plan {
                     result.acts.push(act);
                 }
                 IOPlanElement::Leg(io_leg) => match routing_mode {
-                    RoutingMode::AdHoc => {
+                    UsePlans => {
```

Comment:
This looks weird because one of the branches has the `RoutingMode` identifier and the other doesn't.

Reply:
fixed it in #48